web/src/test/auto/dom/web-test-runner.config.mjs

```diff
@@ -31,34 +31,34 @@ export default {
       name: 'engine/attachment',
       // Relative, from the containing package.json
       files: [
-        'src/test/auto/dom/cases/attachment/**/*.spec.html',
-        'src/test/auto/dom/cases/attachment/**/*.spec.ts'
+        'build/test/dom/cases/attachment/**/*.spec.html',
+        'build/test/dom/cases/attachment/**/*.spec.mjs'
```

Comment:
To compare with the comment I made for keyboard-processor tests:  note how the `build/test/dom/**` pattern is being used here, with minimal changes to boot.